curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md

```diff
@@ -28,6 +28,13 @@ Fulfill the user stories and pass all the tests below to complete this project.
 
 # --hints--
 
+You should link your stylesheet to your HTML file.
+
+```js
+const link = document.querySelector('.fcc-injected-styles');
+assert.exists(link);
+```
+
```

Comment:
We should mirror this change in the legacy project file too.

Reply:
🤦‍♂️